spinalcordtoolbox/download.py

```diff
@@ -159,7 +159,13 @@
             "https://github.com/ivadomed/multimodal-registration/releases/download/r20220512/models.zip"
         ],
         "default_location": os.path.join(__sct_dir__, "data", "deepreg_models"),
-    }
+    },
+    "data_normalize_metric_pam50": {
+        "mirrors": [
+            "https://github.com/spinalcordtoolbox/data_normalize_metric_pam50/archive/refs/tags/r20230221.zip"
+        ],
+        "default_location": os.path.join(__sct_dir__, "data", "data_normalize_metric_pam50"),
+    },
 }
 
 
```

Comment:
Side-question: Would it be beneficial to combine this PR with #4003?

The reason I suggest this is because of the following scenario:

- We merge this PR to master.
- [#4003](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003) is updated with master to include the new dataset link.
- We start playing around with [#4003](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003) to compute compression metrics from the CSV dataset.
- During testing, we identify some flaw or concern with the dataset (perhaps some issue with how the metrics were generated via [#3977](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977)).
- We need to regenerate the dataset and create a new release link, and thus a new change to `master`.

By comparison, if this dataset link were to be included as part of #4003, then it could potentially save a bit of clutter on master, and keep related development grouped in the same location, since any changes to the dataset would be part of #4003's development.

(But, maybe the scenario I describe is unlikely, or easy enough to handle if something goes wrong?)

Reply:
yes, we can include this in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003, I agree!